db/migrate/20190705091659_delete_unnecessary_proceeding_types.rb

```diff
@@ -0,0 +1,26 @@
+class DeleteUnnecessaryProceedingTypes < ActiveRecord::Migration[5.2]
+  def safe_proceeding_id
+    ProceedingType.find_by(ccms_code: 'DA001').id
+  end
+
+  def proceeding_ids_to_be_deleted
+    ProceedingType.where.not(ccms_code: %w[DA001 DA002 DA003 DA004 DA005 DA006 DA007]).pluck(:id)
+  end
+
+  def up
+    ApplicationProceedingType.where(proceeding_type_id: proceeding_ids_to_be_deleted).update_all(proceeding_type_id: safe_proceeding_id)
+    ProceedingType.delete(proceeding_ids_to_be_deleted)
+    ProceedingType.create(code: 'PR0220',
```

Comment:
`.create!` instead of `.create`

Reply:
Anyway, I don't think we need to add this proceeding type here. I think that it will be added by `db:seed` which is run when the container starts: https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/docker/setup_db#L8

Reply:
Thanks. I tried it using just the additional seed entry but that didn't work for me, so I added it to the migration too. I've removed it from the migration and it works now though, I must have been doing something wrong 🤷‍♂